Cargo.toml

```diff
@@ -26,6 +26,9 @@ tempfile = "3.2.0"
 
 [workspace]
 members = [
+	"bridges/primitives/kusama",
+	"bridges/primitives/polkadot",
+	"bridges/primitives/runtime",
```

Comment:
This reminds me - we have a branch called `bridges-integration` which already adds some Polkadot and Kusama primitives. While it's a bit outdated (last commit was Dec 22nd), we should probably use it as a reference for integration moving forward.

Reply:
Yes, I'll add integration in a separate PR cause it should be properly reviewed. This one is 65k lines of code that was already reviewed in bridges repo, hence the separation.